### DISCUSSION TITLE
Hover Feature on Issues Detail Page will not go away

### DISCUSSION THREAD

**Discussion by dagnum** [community/community#44947] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

When on the Issues Detail Page the hover feature will not go away even when you move the mouse away and even when you click.  The only way it goes away is if you hover over something else then that one wont go away!  Driving me nutts.

I am on Windows 10 FireFox LATEST

**Comment by radar-blip** [comment, marked as answer]
GitHub is aware of this and are working on a fix.
https://github.com/orgs/community/discussions/44630#discussioncomment-4729878

**Comment by dagnum** [comment]
Thank you!!
